ui/src/reusable_ui/components/page_layout/Page.tsx

```diff
@@ -0,0 +1,36 @@
+// Libraries
+import React, {Component} from 'react'
+import classnames from 'classnames'
+
+// Components
+import PageHeader from 'src/reusable_ui/components/page_layout/PageHeader'
+import PageTitle from 'src/reusable_ui/components/page_layout/PageHeaderTitle'
+import PageContents from 'src/reusable_ui/components/page_layout/PageContents'
+
+import {ErrorHandling} from 'src/shared/decorators/errors'
+
+interface Props {
+  children: JSX.Element[] | JSX.Element
+  className?: string
+}
+
+@ErrorHandling
+class Page extends Component<Props> {
+  public static Header = PageHeader
+  public static Title = PageTitle
+  public static Contents = PageContents
+
+  public render() {
+    return <div className={this.className}>{this.props.children}</div>
+  }
+
+  private get className(): string {
+    const {className} = this.props
+
+    return classnames('page', {
+      [`${className}`]: className,
```

Comment:
I've used this approach a few different places, but essentially trying to print the value of the prop rather than the name of the prop so long as it exists. I think in this specific instance it looks weird cuz there's only the one option in classnames. Here's an example from a different component:

```
return classnames('form--element form--submit', {
      [`col-xs-${colsXS}`]: colsXS,
      [`col-sm-${colsSM}`]: colsSM,
      [`col-md-${colsMD}`]: colsMD,
      [`col-lg-${colsLG}`]: colsLG,
      [`col-xs-offset-${offsetXS}`]: offsetXS,
      [`col-sm-offset-${offsetSM}`]: offsetSM,
      [`col-md-offset-${offsetMD}`]: offsetMD,
      [`col-lg-offset-${offsetLG}`]: offsetLG,
    })
```

I suppose it is kind of like this: https://github.com/JedWatson/classnames#dynamic-class-names-with-es2015

Reply:
ah that makes a lot more sense in context of more classnames. cool.